pages/groupe-3.html

```diff
@@ -3,6 +3,70 @@
     <title>Groupe 3 : Wendy Salaun - Yousra Khalfat - Clémence Luszcz - Louise Larroque</title>
 </head>
 <body>
-    <p><a href="../index.html">Retour au menu</a></p>
+<h1> La mode se démode, le style jamais
+    <h2>
+        <p> <acronym> Comment s'habiller pour un festival de musique ?
+
+            <h3> Un spécial festival tordant le cou aux clichés et célébrant une bohème qui a de la trempe...
+                Et du sex-appeal. Let's go to We Love Green.
```

Comment:
Et du <span lang="en">sex-appeal.</span>  <span lang="en">Let's go to We Love Green.</span> </h3>